src/Tests/Microsoft.NET.Build.Tests/GlobalPropertyFlowTests.cs

```diff
@@ -86,7 +86,7 @@ public void TestGlobalPropertyFlowToLibrary(bool passSelfContained, bool passRun
 
             bool buildingSelfContained = passSelfContained || passRuntimeIdentifier;
```

Comment:
This should probably renamed or the logic reworked, since passing a RuntimeIdentifier doesn't imply that the project will be self-contained.  Same with other tests here.

Reply:
I changed it to `appIsSelfContainedOrRuntimeSpecific` which is longer and could be better, but I think it does make it clearer